src/sonic-host-services/scripts/hostcfgd

```diff
@@ -987,18 +1013,25 @@ class HostConfigDaemon:
         # Initialize AAACfg
         self.hostname_cache=""
         self.aaacfg = AaaCfg()
-        
 
-    def load(self):
-        aaa = self.config_db.get_table('AAA')
-        tacacs_global = self.config_db.get_table('TACPLUS')
-        tacacs_server = self.config_db.get_table('TACPLUS_SERVER')
-        radius_global = self.config_db.get_table('RADIUS')
-        radius_server = self.config_db.get_table('RADIUS_SERVER')
+    def load(self, init_data):
+        features = init_data['FEATURE']
+        aaa = init_data['AAA']
+        tacacs_global = init_data['TACPLUS']
+        tacacs_server = init_data['TACPLUS_SERVER']
+        radius_global = init_data['RADIUS']
+        radius_server = init_data['RADIUS_SERVER']
+        lpbk_table = init_data['LOOPBACK_INTERFACE']
+        ntp_server = init_data['NTP_SERVER']
+        ntp_global = init_data['NTP']
+
+        self.feature_handler.update_all_features_config(features)
```

Comment:
>update_all_features_config [](http://example.com/codeflow?start=29&length=26)

Why not use `sync_state_field()` directly? No need to add new function.

Reply:
Hrm I was reverting to the prior load function in the old code, let me compare these and see if thats doable.

Reply:
Works.